dask_expr/_shuffle.py

```diff
@@ -82,11 +82,13 @@ class Shuffle(Expr):
         "ignore_index",
         "backend",
         "options",
+        "index_shuffle",
```

Comment:
It would be helpful to document this operand as well.

Reply:
yep